docs/fields.asciidoc

```diff
@@ -38,6 +38,39 @@ Fields common to various APM events.
 
 
 
+*`data_stream.type`*::
++
+--
+Data stream type, e.g. logs, metrics, or traces.
```

Comment:
I'd be a bit more explicit and state that can only be one of those values, and not an arbitrary string

Reply:
Reworded to clarify that it will be one of those values.